packages/pools/src/utils/boosted/apr/fetchAlpBoostedPoolApr.ts

```diff
@@ -0,0 +1,63 @@
+import BigNumber from 'bignumber.js'
+import { PublicClient } from 'viem'
+
+const WEEK = 7
+const YEAR = 365
+const CONTRACT_ADDRESS = '0xB3879E95a4B8e3eE570c232B19d520821F540E48'
```

Comment:
We can deal with this later but the proper way is we have a map from chainId to contract addresses. So the function call of `fetchAlpBoostedPoolApr(chainId, client)` is completely chain neutral

Reply:
// TODO